### PR TITLE
UI - Rental Unit Address Breakout

### PR DESCRIPTION
**Pull request by shaangill025**
*Issue:*

- https://github.com/bcgov/entity/issues/24125

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by dimak1** [comment]
@shaangill025 before I review it, I think it should be merged into Lekshmi's branch, so it can be tested before merging to main. I think that was the idea. cc: @kris-daxiom

File: strr-web/interfaces/account-i.ts [hunk 1]
```diff
@@ -174,7 +174,15 @@ export interface MailingAddressAPII {
   country: string
 }
 
-export interface UnitAddressI extends MailingAddressAPII {
+export interface UnitAddressI {
```
[the remark this answers is not in the document]
Comment (shaangill025) [reply strr-web/interfaces/account-i.ts:177]: Will address this in #293

File: strr-web/enums/property-type-map-e.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+export enum PropertyTypeValueMapE {
```
[the remark this answers is not in the document]
Comment (shaangill025) [reply strr-web/enums/property-type-map-e.ts:1]: Will address this in #293

File: strr-web/interfaces/registration-i.ts [hunk 3]
```diff
@@ -3,11 +3,22 @@ export interface RegistrationAddressI {
...
   province: string
-  address: string
+  streetNumber: string
+  streetName: string
+  unitNumber?: string
   addressLineTwo: string
   nickname: string
 }
 
+export interface MailingAddressI {
```
[the remark this answers is not in the document]
Comment (shaangill025) [reply strr-web/interfaces/registration-i.ts:13]: Will address this in #293

File: strr-web/stores/strr.ts [hunk 4]
```diff
@@ -251,25 +251,89 @@ export const propertyDetailsSchema = z.object({
...
+  businessLicenseExpiryDate: requiredNonEmptyString
+})
+
+const businessLicenseExpiryRequiredSchema = basePropertyDetailsSchema.extend({
+  _hasLicense: z.literal(false),
+  businessLicense: z.string().length(0),
+  businessLicenseExpiryDate: optionalOrEmptyString
+})
+
+const businessLicensePropertyDetailsSchema = z.object({
```
[the remark this answers is not in the document]
Comment (shaangill025) [reply strr-web/stores/strr.ts:270]: Will address this in #293

**Pull request closed without merging**